webapp/src/main/resources/application.sql.yml

```diff
@@ -23,3 +23,17 @@ sql:
   status:
     readTest: >-
       SELECT ist.name FROM import_status ist
+
+  student_group_batch:
+    create: >-
+      INSERT INTO upload_student_group_batch (status, digest, creator, message)
+        VALUES (:status, :digest, :creator, :message)
+
+    findOne: >-
+      SELECT * FROM upload_student_group_batch WHERE id=:id
+
+    findOneByDigest: >-
+      SELECT * FROM upload_student_group_batch WHERE digest=:digest LIMIT 1
```

Comment:
So digest is not unique in import because of replay, i'm not sure we want that for upload_student_group_batch because we do not want to process the same files twice.  But if there is a code or system error and we need to process the file again we need a way to process it again but nut upload it again?